auth-api/src/auth_api/services/org.py

```diff
@@ -828,15 +828,11 @@ def approve_or_reject(org_id: int, is_approved: bool, token_info: Dict, origin_u
 
         org.save()
 
-        if org.status_code == OrgStatus.PENDING_STAFF_REVIEW.value and \
-                org.access_type in (AccessType.EXTRA_PROVINCIAL.value, AccessType.REGULAR_BCEID.value):
-            # Find admin email address
-            admin_email = ContactLinkModel.find_by_user_id(org.members[0].user.id).contact.email
+        # Find admin email address
+        admin_email = ContactLinkModel.find_by_user_id(org.members[0].user.id).contact.email
+        if org.access_type in (AccessType.EXTRA_PROVINCIAL.value, AccessType.REGULAR_BCEID.value):
             Org.send_approved_rejected_notification(admin_email, org.name, org.id, org.status_code, origin_url)
-
-        elif org.status_code == OrgStatus.PENDING_STAFF_REVIEW.value and \
-                org.access_type == AccessType.GOVM.value:
-            admin_email = ContactLinkModel.find_by_user_id(org.members[0].user.id).contact.email
+        else:
```

Comment:
may be do an elif access_type=govm ; so that if somebody use this for another org ,it wont be sending a govm mail